src/Pos/Genesis.hs

```diff
@@ -107,19 +109,19 @@ bitcoinDistribution20 :: [Coin]
 bitcoinDistribution20 = map mkCoin
     [200,163,120,105,78,76,57,50,46,31,26,13,11,11,7,4,2,0,0,0]
 
-stakeDistribution :: StakeDistribution -> [Coin]
+stakeDistribution :: StakeDistribution -> [(Coin, [(StakeholderId, Coin)])]
```

Comment:
Looks like `[(StakeholderId, Coin)]` is `[]` in all cases. Probably that's because I am stupid and don't understand something. Can you clarify?